android/src/app/organicmaps/bookmarks/BookmarkCategoriesAdapter.java

```diff
@@ -291,7 +291,7 @@ public void onClick(View v)
     {
       BookmarkCategory category = mHolder.getEntity();
       BookmarkManager.INSTANCE.toggleCategoryVisibility(category);
-      notifyItemChanged(mHolder.getAdapterPosition());
+      notifyItemChanged(mHolder.getBindingAdapterPosition());
```

Comment:
What is the difference with [getAbsoluteAdapterPosition](https://developer.android.com/reference/androidx/recyclerview/widget/RecyclerView.ViewHolder#getAbsoluteAdapterPosition()) ? Shouldn't it be used instead of getBindingAdapterPosition in some cases?

Reply:
Have a look at the source I have mentioned in this PR.
It explains the difference between the two.

Reply:
Thanks, I've already read this documentation before I wrote the question.

Reply:
So you confirm that 
> or if you want the position as [RecyclerView](https://developer.android.com/reference/androidx/recyclerview/widget/RecyclerView) sees it, you should call [getAbsoluteAdapterPosition](https://developer.android.com/reference/androidx/recyclerview/widget/RecyclerView.ViewHolder#getAbsoluteAdapterPosition()).

is not our case in all these lines touched in this PR, correct?

Reply:
Yeah right.
We don't require getAbsoluteAdapterPosition in any of the instances I changed.